packages/smart-wallet/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@agoric/assert": "^0.6.0",
     "@agoric/casting": "^0.4.2",
+    "@agoric/deploy-script-support": "^0.10.3",
```

Comment:
this is only here for `upgrade-wallet-factory2.js` in smart-wallet/proposals. But it's not a proposal; it's a builder. Please move to `packages/builders/scripts/smart-wallet`
```suggestion
```